docs/analytics_tools/tools_quick_links.md

```diff
@@ -7,16 +7,10 @@
 | Tool                                                                                               | Purpose                                 |
 | -------------------------------------------------------------------------------------------------- | --------------------------------------- |
 | [**Analytics Repo**](https://github.com/cal-itp/data-analyses)                                     | Analytics team code repository.         |
-| [**Analytics Project Board**](https://github.com/cal-itp/data-analyses/projects/1)                 | Analytics team work management.         |
+| [**Analytics Project Board**](https://github.com/cal-itp/data-analyses/projects/1)                 | Analytics team list of active issues.   |
 | [**notebooks.calitp.org**](https://notebooks.calitp.org/)                                          | JupyterHub cloud-based notebooks        |
 | [**dashboards.calitp.org**](https://dashboards.calitp.org/)                                        | Metabase dashboards & Business Insights |
 | [**dbt-docs.calitp.org**](https://dbt-docs.calitp.org/)                                            | DBT warehouse documentation             |
 | [**analysis.calitp.org**](https://analysis.calitp.org/)                                            | Analytics portfolio landing page        |
 | [**Google BigQuery**](https://console.cloud.google.com/bigquery)                                   | Our warehouse and SQL Querying          |
 | [**Google Cloud Storage**](https://console.cloud.google.com/storage/browser/calitp-analytics-data) | Cloud file storage                      |
-
-&#160;
-
-```{admonition} Still need access to a tool on this page?
-Ask in the `#services-team` channel in the Cal-ITP Slack.
```

Comment:
This is still protocol as far as I know! this slack channel still exists. But Evan is also a resource for tooling access (and I can grant access to some tools as well). Let's post in slack saying that we're updating the docs and we'd like to know if there's specific protocol we should put in, or not.

Reply:
Should I post on #services-team channel or data-office-hours?